lumispy/utils/axes.py

```diff
@@ -149,10 +149,10 @@ def axis2invcm(ax0):
         raise AttributeError(r"Signal unit is already cm$^{-1}$.")
     # transform axis, invert direction
     if ax0.units == "µm":
-        invcmaxis = nm2invcm(1000 * ax0.axis)[::-1]
+        invcmaxis = nm2invcm(1000 * ax0.axis)[::-1].astype("float64")
```

Comment:
Wouldn't `float` be better (to account for x32bit computers too)?

Reply:
Indeed, just took what the working format was locally - but the more general one would indeed be better.